### PR TITLE
fix(transaction-summary): Do not pass project to issue search query

### PR DESCRIPTION
**Pull request by Zylphrex**
Issue search currently does not support the project filter in the search syntax.
This filters out the project tag from the filter since the project is already
specified in the project param in the related issues query.

### CODE REVIEW

File: static/app/views/performance/transactionSummary/relatedIssues.tsx [hunk 1]
```diff
@@ -55,9 +55,11 @@ class RelatedIssues extends Component<Props> {
     });
     currentFilter.addQuery('is:unresolved').setTagValues('transaction', [transaction]);
 
-    // Filter out key_transaction from being passed to issues as it will cause an error.
-    currentFilter.removeTag('key_transaction');
-    currentFilter.removeTag('team_key_transaction');
+    // Filter out these search terms from being passed to issues as they will cause an error.
+    currentFilter
+      .removeTag('key_transaction')
+      .removeTag('team_key_transaction')
```
Comment (wmak) [inline static/app/views/performance/transactionSummary/relatedIssues.tsx:61]: should we add these two to `TRACING_FIELDS`?

**Review by wmak — APPROVED** [review]
lgtm

File: static/app/views/performance/transactionSummary/relatedIssues.tsx (hunk 1 above)
Comment (Zylphrex) [reply static/app/views/performance/transactionSummary/relatedIssues.tsx]: Good call, I've moved them into `TRACING_FIELDS`. They **will not** appear as an option in discover

Comment (wmak) [reply static/app/views/performance/transactionSummary/relatedIssues.tsx]: 🙏 thanks for checking that

**Pull request merged**